scripts/on-branch-created.js

```diff
@@ -0,0 +1,77 @@
+//# sourceURL=on-branch-created.js
+'use strict';
+
+//  Description
+//
+// This script work as a handler that checks if an issue is in open status.
```

Comment:
I've slightly re-phrased the description:

//  This script does an automatic issue transition to "In progress" status if
// the issue is still in "Open" status when a branch has been created for the issue.
// It works on the assumption that an issue key is present in the name of the created branch.
// First issue's key from the created branch name will be taken as an issue for the transition.
// Issue's status will be changed on behalf of the issue's assignee.